docs/guide/ai-extension-install.md

```
@@ -1,87 +1,99 @@
-1. 访问 [v0.0.7 下载地址](https://docs.opentiny.design/download/extension-0.0.7.zip) 下载 `ai-extension.zip` 到本地。
-2. 将其解压到一个固定的文件夹中（安装后请勿移动或删除该文件夹）。
+1. 访问 [下载地址](https://docs.opentiny.design/download/extension-0.0.7.zip) 下载扩展压缩包到本地。
+2. 将其解压到一个**固定的文件夹**（安装后请勿移动或删除该文件夹）。
 
 ### 步骤二：开启开发者模式
+
 1. 打开浏览器的扩展管理页面（Chrome 地址栏输入 `chrome://extensions/`）。
-2. 开启右上角的“开发者模式” (Developer mode)。
+2. 开启右上角的"开发者模式"（Developer mode）。
 
 ![](../assets/images/extension/5.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)